plugins/PerObjectSettingsTool/PerObjectSettingsPanel.qml

```diff
@@ -395,11 +394,33 @@ Item {
                 // Set skip setting, it will prevent from resetting selected mesh_type
                 contents.model.visibilityHandler.addSkipResetSetting(meshTypeSelection.model.get(meshTypeSelection.currentIndex).type)
                 listview.model.forceUpdate()
+
+                updateFilter()
+            }
+        }
+
+        function updateFilter()
+        {
+            var new_filter = {};
+            if (printSequencePropertyProvider.properties.value == "one_at_a_time")
+            {
+                new_filter["settable_per_meshgroup"] = true;
             }
+            else
+            {
+                new_filter["settable_per_mesh"] = true;
+            }
+
+            if(filterInput.text != "")
+            {
+                new_filter["i18n_label"] = "*" + filterInput.text;
```

Comment:
Doesn't there also need to be an asterisk after the input? Like:

    "*" + filterInput.text + "*"

Reply:
SettingView and SettingVisibilityPage both filter with a single prepended "*".